tests/lib/rules/indent.js

```diff
@@ -289,7 +291,20 @@ eslintTester.addRuleTest("lib/rules/indent", {
                 "        break;\n" +
                 "}",
             options: [4, {indentSwitchCase: true}],
-            errors: expectedErrors([[13, 4], [15, 4], [17, 4]])
+            errors: expectedErrors([[5, 8], [8, 8]])
+        },
+        {
+            code:
+                "switch (a) {\n" +
+                "case '1':\n" +
+                "b();\n" +
+                "break;\n" +
+                "default:\n" +
+                "c();\n" +
+                "break;\n" +
+                "}",
+            options: [4, {indentSwitchCase: true}],
+            errors: expectedErrors([[3, 4], [4, 4], [6, 4], [7, 4]])
```

Comment:
That's why I asked for clarification - I also prefer this style, but a lot of people prefer the opposite (same level with `switch`) and currently ESLint supports both; so I'm not sure whether this rule+option affects only bodies of `case` or should it change behavior of `case` itself as well.